src/index.html

```diff
@@ -7,6 +7,640 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#">
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
```

Comment:
```suggestion
      <nav class="nav">
        <ul class="nav__list">
          <li class="nav__item">
            <a class="nav__link is-active" href="#">Apple</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Samsung</a>
          </li>
          
          <li class="nav__item">
            <a class="nav__link" href="#">Smartphones</a>
          </li>
          
          <li class="nav__item">
            <a
              class="nav__link"
              href="#"
              data-qa="nav-hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="nav__item">
```
<img width="1080" alt="Screenshot 2022-12-07 at 13 52 19" src="https://user-images.githubusercontent.com/104769168/206172644-fc7c269a-efa0-4138-abcc-003c4b3b4bfe.png">
fix code style all over the page